modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -896,7 +896,11 @@ const FrontendDataSet = ({
 						/>
 					)}
 
-					<div className="data-set-wrapper" ref={wrapperRef}>
+					<div
+						className="data-set-wrapper"
+						data-testid={`visualization-mode-${activeView.name}`}
```

Comment:
Use `data-testid` to reference the visualization mode as it main use is testing.

Reply:
Just asking, is there any alternative for a `testid` here (for not losing a chance to test accessibility at the same time) ?